extensionsGallery-insider.json

```diff
@@ -3358,6 +3358,67 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "69",
+					"extensionName": "code-settings-sync",
+					"displayName": "Settings Sync",
+					"shortDescription": "Synchronize Settings, Snippets, Themes, File Icons, Launch, Keybindings, Workspaces and Extensions Across Multiple Machines Using GitHub Gist.",
+					"publisher": {
+						"displayName": "Shan Khan",
+						"publisherId": "Shan",
+						"publisherName": "Shan"
+					},
+					"versions": [
+						{
+							"version": "3.4.3",
+							"lastUpdated": "9/23/2019",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
```

Comment:
for 3rd-party extensions we can't use the direct in-product installation due to internal policy.  users will need to download the VSIX and side-load.